tutorials/Tutorial17_Audio.py

```diff
@@ -0,0 +1,123 @@
+    # with the addition of a node that converts our answers into audio files!
+    #
+    # Note that if you're doing document retrieval, Haystack provides a DocumentToSpeech node
+    # that can be used in a similar way (or in an indexing pipeline).
+
+    retriever = BM25Retriever(document_store=document_store)
+    reader = FARMReader(model_name_or_path="deepset/roberta-base-squad2", use_gpu=True)
+    answer2speech = AnswerToSpeech(
+        model_name_or_path="espnet/kan-bayashi_ljspeech_vits",
+        generated_audio_dir=Path(__file__).parent / "audio_answers",
+    )
+
+    audio_pipeline = Pipeline()
+    audio_pipeline.add_node(retriever, name="Retriever", inputs=["Query"])
+    audio_pipeline.add_node(reader, name="Reader", inputs=["Retriever"])
+    audio_pipeline.add_node(answer2speech, name="AnswerToSpeech", inputs=["Reader"])
+
+    prediction = audio_pipeline.run(
+        query="Who is the father of Arya Stark?", params={"Retriever": {"top_k": 10}, "Reader": {"top_k": 5}}
+    )
+
+    # Now you can either print the object directly
+    print("\n\nRaw object:\n")
+    from pprint import pprint
+
+    pprint(prediction)
+
+    # Sample output:
+    # {
+    #     'answers': [ <Answer: answer='Eddard', type='extractive', score=0.9919578731060028, offsets_in_document=[{'start': 608, 'end': 615}], offsets_in_context=[{'start': 72, 'end': 79}], document_id='cc75f739897ecbf8c14657b13dda890e', meta={'name': '454_Music_of_Game_of_Thrones.txt'}}, context='...' >,
+    #                  <Answer: answer='Ned', type='extractive', score=0.9767240881919861, offsets_in_document=[{'start': 3687, 'end': 3801}], offsets_in_context=[{'start': 18, 'end': 132}], document_id='9acf17ec9083c4022f69eb4a37187080', meta={'name': '454_Music_of_Game_of_Thrones.txt'}}, context='...' >,
+    #                  ...
+    #                ]
+    #     'documents': [ <Document: content_type='text', score=0.8034909798951382, meta={'name': '332_Sansa_Stark.txt'}, embedding=None, id=d1f36ec7170e4c46cde65787fe125dfe', content='\n===\'\'A Game of Thrones\'\'===\nSansa Stark begins the novel by being betrothed to Crown ...'>,
+    #                    <Document: content_type='text', score=0.8002150354529785, meta={'name': '191_Gendry.txt'}, embedding=None, id='dd4e070a22896afa81748d6510006d2', 'content='\n===Season 2===\nGendry travels North with Yoren and other Night's Watch recruits, including Arya ...'>,
+    #                    ...
+    #                  ],
+    #     'no_ans_gap':  11.688868522644043,
+    #     'node_id': 'Reader',
+    #     'params': {'Reader': {'top_k': 5}, 'Retriever': {'top_k': 5}},
+    #     'query': 'Who is the father of Arya Stark?',
+    #     'root_node': 'Query'
+    # }
+
+    # Or use a util to simplify the output
+    # Change `minimum` to `medium` or `all` to raise the level of detail
+    print("\n\nSimplified output:\n")
+    print_answers(prediction, details="minimum")
```

Comment:
Now that we also have `DocumentToSpeech` I was wondering whether we could exemplify its usage also in the tutorial. For example, at the end of the tutorial show how to convert the document that contains the answer to speech outside of the pipeline, just use the node separately, and play the sound. That would make more users aware of `DocumentToSpeech`. What do you think?

Reply:
Oh right that's a nice idea. I didn't use it because I didn't want to make another whole pipeline for document search :smile: but this is a good way to use it, even though it won't be in a pipeline. Still better than not using it at all :grin: I'll add!

Reply:
Or maybe I can add it to the indexing pipeline! :eyes: I'll have a try.